public/backend.js

```diff
@@ -20,7 +21,11 @@ io.on('connection', (socket) => {
       console.log('user disconnected');
   });
   socket.on('chat message', (msg) => {
-      messages.push(msg);
+      messages.push([
```

Comment:
Weird that an array works but an object doesn't (as an array IS an object). Will investigate.

Reply:
I probably just borked it somehow.